CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # go-ipfs changelog
 
-### 0.4.10 - 2017-06-18
+### 0.4.10 - 2017-06-27
+
+Ipfs 0.4.10 is a patch release that contains several exciting new features,
```

Comment:
Hrm... we do tend to stick with unified case. But the previous changelog updates have capitalized it this way. Though theres no time like the present to change. Do you think it should be all lowercase here?